### PR TITLE
feat: add @fastify/htmx version of the frontend app

### PR DESCRIPTION
**Pull request by galvez**

### CODE REVIEW

File: apps/movie-quotes-api/quotes/quotes.cjs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+'use strict'
```
Comment (mcollina) [inline apps/movie-quotes-api/quotes/quotes.cjs:1]: why was this client generated in this folder?

Comment (galvez) [reply apps/movie-quotes-api/quotes/quotes.cjs:1]: Oh, that's a leftover — removing...

File: apps/movie-quotes-frontend-htmx/pnpm-lock.yaml
Comment (marcopiraccini) [inline apps/movie-quotes-frontend-htmx/pnpm-lock.yaml:1]: Are we using here BOTH `package-lock` and `pnpm-lock`? I think we should stick to `npm`, if we change to `pnpm` we need to do the same in `api` and the other frontend, and update the README.

**Review by marcopiraccini — CHANGES_REQUESTED** [review]
Looks very good, tnx! Just check my comment on `pnpm-lock`

**Comment by galvez** [comment]
Standardized on npm — also pushed a small fix.

Can you merge please @marcopiraccini or @mcollina

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**